src/components/LandscapeVideoCard/LandscapeVideoCard.jsx

```diff
@@ -18,9 +20,20 @@ function LandscapeVideoCard({
   youtubeId,
   category,
   _id,
+  inHistory,
+  inLikedVideo,
 }) {
   const { authState } = useAuth();
   const { historyDispatch } = useHistory();
+  const { likeDispatch } = useLike();
+  const deleteFromList = () => {
+    if (!!inHistory) {
+      deleteFromHistory(_id, authState.token, historyDispatch);
+    }
+    if (!!inLikedVideo) {
+      deleteFromLiked(_id, authState.token, likeDispatch);
+    }
+  };
```

Comment:
Just curious, if `(!!inLikedVideo)` why have you used the Logical Not operator in this way?

Reply:
It is actually redundant as it coerces to a boolean.
I will remove it